inspec.yml

```diff
@@ -23,7 +23,8 @@ inputs:
     required: true
     sensitive: true
 
-  - name: pg_owner_password
+# for possible future use
+  - name: pg_owner_password 
     description: "The postgres database owner password"
     type: string
     value: ""

```

Comment:
The entire section of an unused input can be commented out

Reply:
Done